test/blackbox-tests/test-cases/pkg/env-conditional-dependencies.t

```diff
@@ -33,16 +33,14 @@ Create a workspace config that defines separate build contexts for macos and lin
   >  (default
   >   (name linux)
   >   (lock dune.linux.lock)
-  >   (solver_env
-  >    (sys
-  >     (os linux)))))
+  >   (override_solver_sys_vars
```

Comment:
I think my concern would that these names are getting increasingly complex and hard to understand. A user doesn't really know (nor should they really need to care) what solver sys vars are.

Reply:
But should a user be reading this in the first place?